config/channel/assisted-deployment-ci/bug_master_configuration.yaml

```diff
@@ -95,6 +89,8 @@ assignees:
     - jobs:
         - e2e-ai-operator-ztp-disconnected-periodic
         - e2e-metal-assisted-day2-periodic
+        - e2e-ai-operator-ztp-hypershift-zero-nodes-periodic
+        - e2e-metal-assisted-kube-api-late-binding-single-node-periodic
```

Comment:
e2e-metal-assisted-kube-api-late-binding-single-node should be on Paul and not derez

Reply:
`e2e-metal-assisted-kube-api-late-binding-single-node-periodic` is registered to derez and not to Paul, I'll update it in the doc